projects/pgai/db/sql/idempotent/014-semantic-catalog.sql

```diff
@@ -0,0 +1,1293 @@
+
+-------------------------------------------------------------------------------
+-- embedding_sentence_transformers
+create or replace function ai.embedding_sentence_transformers
```

Comment:
With this it looks like we're adding another function to specify embedding providers for vectorizers, but this configuration is not supported by pgai vectorizer, which is confusing.

Reply:
we wanted an option that has no external dependencies. maybe we should add this to pgai vectorizer